### PR TITLE
Adding a new footer component

### PR DESCRIPTION
**Pull request by SanjeebNath382**
## What type of PR is this? (check all applicable)

- [ ] ♻️ Refactor
- [x] ✨ Feature
- [ ] 🐛 Bug Fix
- [ ] 👷 Optimization
- [ ] 📝 Documentation Update
- [ ] 🚩 Other

## Added to documentation?

- [ ] 📜 readme
- [x] 🙅 no documentation needed

## [optional] What gif best describes this PR or how it makes you feel?

### CODE REVIEW

File: components/footer-simple.html [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+            <a class="text-gray-600 hover:text-gray-800">Third Link</a>
+          </li>
+          <li>
+            <a class="text-gray-600 hover:text-gray-800">Fourth Link</a>
+          </li>
+        </nav>
+      </div>
+    </div>
+  </div>
+  <div class="bg-red-200">
```
Comment (thinkverse) [inline components/footer-simple.html:74]: Not sure about the red color there, seems out of place for me.

File: components/footer-simple.html [hunk 2]
```diff
@@ -0,0 +1,103 @@
+<footer class="text-gray-700 body-font">
```
Comment (thinkverse) [inline components/footer-simple.html:1]: Maybe just add an overall background to the whole footer instead?

```suggestion
<footer class="text-gray-700 bg-gray-400 body-font">
```

**Review by thinkverse — CHANGES_REQUESTED** [review]
Thank you for contributing to Tails again @SanjeebNath382 🙂

I would suggest wrapping the footer in a div and just pushing it to the bottom for presentational purposes, something like `<div class="bg-gray-100 flex w-full min-h-screen items-end">` and then adding a `w-full` to the footer itself to fill out the full width.

Also, some spacing on the links would be appreciated. They look a bit cramped.

Not 100% on the mobile view, there isn't enough space on the social icons, and the text isn't aligned correctly, I would spend some more times on the mobile view. 🙂

**Comment by connecteev** [comment]
@SanjeebNath382 Any chance you could attach some screenshots?

**Comment by thinkverse** [comment]
This PR now contains more than one component and none of the suggested changes has been implemented. Therefore I'm closing this PR and suggest you separate the component into separate PRs.

**Pull request closed without merging**